installing/installing_vsphere/ipi/ipi-vsphere-installation-reqs.adoc

```diff
@@ -17,6 +17,7 @@ include::modules/vmware-csi-driver-reqs.adoc[leveloffset=+1]
 
 * To remove a third-party vSphere CSI driver, see xref:../../../storage/container_storage_interface/persistent-storage-csi-vsphere.adoc#persistent-storage-csi-vsphere-install-issues_persistent-storage-csi-vsphere[Removing a third-party vSphere CSI Driver].
 * To update the hardware version for your vSphere nodes, see xref:../../../updating/updating_a_cluster/updating-hardware-on-nodes-running-on-vsphere.adoc#updating-hardware-on-nodes-running-on-vsphere[Updating hardware on nodes running in vSphere].
+* To list of minimum permissions for CSI and its components, see the "Minimum permissions for the storage components" table in the xref:../../installing_vsphere/ipi/ipi-vsphere-installation-reqs#installation-vsphere-installer-infra-requirements_ipi-vsphere-installation-reqs[Required vCenter account privileges] section.
```

Comment:
For this entire additional resources section (not necessarily in the scope of this PR), please see the [guidelines for additional resources sections](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections) in the OCP docs contribution guide. In particular:

> - The links and xrefs in the unordered list must contain minimal-length, human-readable text between the square brackets. This text is often the title of the linked page.
> - You must not include text outside of the square brackets.

This additional resource has a typo in it ("To list of"), so it's worth fixing even if you don't adjust the rest of the section. Because we're talking about a table, it's tricky, but maybe something like this?

```suggestion
* xref:../../installing_vsphere/ipi/ipi-vsphere-installation-reqs#installation-vsphere-installer-infra-requirements_ipi-vsphere-installation-reqs[The "Minimum permissions for the storage components" table in the "Minimum required vCenter account privileges" section]
```

Reply:
OK. I added IDs and now the xref points directly to the table. All works well in the preview.

Reply:
Love it!